app/controllers/v0/search_click_tracking_controller.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+require 'search_click_tracking/service'
```

Comment:
It looks like this may still be a WIP PR. Do you have changes that include request specs?

Reply:
Thanks for pointing this out, I am new to working with the BE and missed these. I am creating these specs now. Thanks!